### PR TITLE
Docs: Added a table in quality-assurance-test-cases.md 

### LINKED ISSUE #471: Create Quality Assurance table for Email Confirmation API
## Description

As a developer,
I need to know what are the test cases for a certain API,
so that I can write new unit tests.

As a tester,
I need to know what are the test cases for a certain API,
so that I can perform manual tests on the API.

The task is to come up and add a quality assurance table to the docs in the `/docs` folder.

### Update [Required]
- [ ] Add test cases for Email Confirmation API

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by pallavisavant**
**Description**

Added table in quality-assurance-test-cases.md which includes all the required test cases for email confirmation API

Fixes #471

**Type of Change:**

- Documentation

 **Checklist:**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
successfully_confirmed_account
account_already_confirmed
Invalid_verification_token

### CODE REVIEW

**Review by vj-codes — CHANGES_REQUESTED** [review]
@pallavisavant congrats on your first contribution to the repo 🎉 
Capitalize letter` v` in test case 2 and 3 and
Please could you add screenshots of all or failed tests atleast in the PR template (not in docs)
It will help to verify and make the tests pass later.
Then you are done:)

**Comment by pallavisavant** [comment]
@vj-codes included respective changes.....pls do review..Thank you.

**Review by vj-codes — APPROVED** [review]
LGTM 🎉

File: docs/quality-assurance-test-cases.md [hunk 1]
```diff
@@ -214,3 +214,14 @@ Only admin users have access to this.
...
 | Create a task if relation state is different than accepted                                | Fail    |
+
+### Confirmation of users email
+
+**Service:** GET /user/confirm_email/{token}
+
+| Test Case                                                                                      | Outcome |
+|------------------------------------------------------------------------------------------------|---------|
+| Verification token entered is one sent on users registered email entered within 24 hrs         | Success |
+| Verification token of already confirmed users account entered                                  | Success |
```
Comment (isabelcosta) [inline docs/quality-assurance-test-cases.md:225]: Interesting 🤔  If the user is already created the API should not do anything. I guess that counts as a success for now :) @pallavisavant great thought

Comment (isabelcosta) [reply docs/quality-assurance-test-cases.md:225]: I guess here the expected behavior should be a message with saying the user is already confirmed. All in all, i think this is fine :)

**Review by isabelcosta — APPROVED** [review]
Great work @pallavisavant 👏 
Thank you for your contribution! 🎉

**Pull request merged**